pkg/kubernetes/views/add-member-role-dialog.html

```diff
@@ -1,6 +1,12 @@
 <modal-dialog>
     <div class="modal-header">
-        <h4 class="modal-title" translatable="yes">Add Member</h4>
+        <h4 class="modal-title" translatable="yes">
```

Comment:
Unfortunately translatable doesn't work on elements that have children. For now we proabably need to wrap just the "Add Member" part in a span and only have the translatable there.